domains/jwoj.json

```
@@ -0,0 +1,10 @@
+{
+  "description": "Hydro Online Judge System",
+  "owner": {
+    "username": "Evansrrr",
+    "email": "enzoraziore@outlook.com"
+  },
+  "record": {
+    "CNAME": "82.156.38.77"
```

```suggestion
    "A": ["82.156.38.77"]
```
This is an IP address. CNAME records must be a hostname. Perhaps you meant to use an A record?

---

yepe